class-two-factor-core.php

```diff
@@ -1821,9 +1823,20 @@ public static function user_two_factor_options( $user ) {
 			$show_2fa_options ? '' : 'disabled="disabled"'
 		);
 
-		wp_nonce_field( 'user_two_factor_options', '_nonce_user_two_factor_options', false );
+		if ( 1 === count( $enabled_providers ) ) {
+			$notices['warning two-factor-warning-suggest-backup'] = esc_html__( 'To prevent being locked out of your account, consider enabling a backup method like Recovery Codes in case you lose access to your primary authentication method.', 'two-factor' );
+		}
 		?>
 		<h2><?php esc_html_e( 'Two-Factor Options', 'two-factor' ); ?></h2>
+		<?php foreach ( $notices as $notice_type => $notice ) : ?>
+		<div class="<?php echo esc_attr( $notice_type ? 'notice inline notice-' . $notice_type : '' ); ?>">
+			<p><?php echo $notice; ?></p>
```

Comment:
I see the values are already escaped early should there be a `wp_kses` here to be safe / should we mention the values are already escaped above?

Reply:
@kasparsd FYI on the question above for you

Reply:
This is now done in 2284d6a37c7f106a1b275900d6d53db675129bbb. We're using `wp_kses_post()` similar to how WP core does it here:

https://github.com/WordPress/WordPress/blob/369a6754887a948830f2107602648e46cb370dfb/wp-admin/includes/class-wp-site-health.php#L2419